src/Microsoft.OpenApi/Models/References/OpenApiTagReference.cs

```diff
@@ -10,21 +12,24 @@ namespace Microsoft.OpenApi.Models.References
     /// <summary>
     /// Tag Object Reference
     /// </summary>
-    public class OpenApiTagReference : OpenApiTag
+    public class OpenApiTagReference : OpenApiTag, IOpenApiReferenceable
```

Comment:
Why does this need to implement IOpenApiReferenceable? None of the other proxy objects do.

Reply:
Apparently it is a bug in every other reference class because the walker expects all references to implement IOpenApiReferencable  /cc @MaggieKimani1